ranger_http/health_check.go

```
@@ -104,3 +142,13 @@ func HealthCheckHandler(services []func() HealthCheckService) httprouter.Handle
 func ElapsedTimeSince(s time.Time) float64 {
 	return float64(time.Since(s)) / float64(time.Second)
 }
+
+//GetHostname - Returns host name or empty string
+func GetHostname() string {
```

`h` will be a empty string in case of error, so if you really want a empty string you can ignore the error with `_`... 
But once you do that, not sure if you really need this function you can directly:
```
hostname, _ := os.Hostname()
```